examples/custom-server/server.js

```diff
@@ -0,0 +1,33 @@
+const { createServer } = require('http')
+const { parse } = require('url')
+const next = require('next')
+const accepts = require('accepts')
+
+const app = next('.', { dev: true })
+const handle = app.getRequestHandler()
+
+app.prepare()
+.then(() => {
+  createServer((req, res) => {
+    const accept = accepts(req)
```

Comment:
@nkzawa can't `handle` have an optional 3rd argument for parsing accept header, so that we just going to have to parse it once?  I can whip up a PR in your branch if you like.